uint32 HeadFront
uint32 HeadMiddle
uint32 HeadRear
uint32 BumperLeft
uint32 BumperRight
